### DISCUSSION TITLE
How to Use EnumFontFamiliesEx?

### DISCUSSION THREAD

**Discussion by seanocali** [halildurmus/win32#625] [Q&A] [answered]
It's the first feature mentioned in the readme. Can anyone give an example of its usage? Apologies my C++ and Win32 aren't up to speed.

**Comment by halildurmus** [comment, marked as answer]
[example/fonts.dart](https://github.com/timsneath/win32/blob/d5fc3391ba57ace9ff0756d2b122fbfc715d5427/example/fonts.dart) demonstrates the use of `EnumFontFamiliesEx`.
